jquery.dynatable.js

```diff
@@ -1548,10 +1552,7 @@
     refreshQueryString: function(urlString, data, settings) {
       var _this = this,
           queryString = urlString.split('?'),
-          path = queryString.shift(),
-          urlOptions;
-
-      urlOptions = this.deserialize(urlString);
+          urlOptions = this.deserialize(queryString);
```

Comment:
I don't think this part actually works, as it will then pass the path to the `this.deserialize()` function which will then try to turn the path into a parameter due to the fact that the `shift()` line was removed. Wouldn't this part just be fixed by changing this line as you have without removing the preceding 3 lines?